.gitignore

```diff
@@ -93,6 +94,7 @@ htmlcov/
 .cache
 nosetests.xml
 coverage.xml
+macropolo
```

Comment:
@anselmbradford whoops this is left over after some pip install strangeness i was seeing locally. will remove it